projects/portal/src/assets/config.js

```diff
@@ -6,13 +6,13 @@ const faucetJpuPort = location.protocol === 'https:' ? 8005 : 8004;
 const faucetUethPort = location.protocol === 'https:' ? 8007 : 8006;
 const faucetEuuPort = location.protocol === 'https:' ? 8009 : 8008;
 
-const domainCauchyEA = 'a.private-test.ununifi.cauchye.net';
-const domainCauchyEB = 'b.private-test.ununifi.cauchye.net';
-const domainCauchyEC = 'c.private-test.ununifi.cauchye.net';
-const domainCauchyED = 'd.private-test.ununifi.cauchye.net';
+const domainCauchyEA = '45.76.96.57';
+const domainCauchyEB = '104.238.151.213';
+const domainCauchyEC = '45.63.121.67';
+const domainCauchyED = '207.148.89.195';
 
-const chainID = 'ununifi-8-private-test';
-const chainName = 'UnUnuFi (Private)';
+const chainID = 'create-validator-debug';
+const chainName = 'UnUnuFi (create-validator-debug)';
 
```

Comment:
Should I not modify this file?
( This Chain "create-validator-debug" is only for debug. )

Reply:
As you said, This "create-validator-debug" is only for debug, I wanna ask you to put it back. I will debug with appropriate network settings.

Reply:
Reverted config.js changes.